samples/graph-appcatalog-lifecycle/nodejs/bots/dialogBot.js

```diff
@@ -0,0 +1,374 @@
+
+        // Save any state changes. The load happened during the execution of the Dialog.
+        await this.conversationState.saveChanges(context, false);
+        await this.userState.saveChanges(context, false);
+    }
+    async GetData(data) {
+        var arr = [];
+        data.value.slice(0, 10).map(element => {
+            arr.push({
+                id: element.id,
+                displayName: element.displayName
+
+            });
+        });
+        console.log("!", arr);
+        return arr;
+    }
+    async cardActivityAsync(context) {
+        const cardActions = [
+            {
+                type: ActionTypes.MessageBack,
+                title: 'Home',
+                value: 1,
+                text: 'home',
+            },
+            {
+                type: ActionTypes.MessageBack,
+                title: 'List',
+                value: 2,
+                text: 'list',
+            },
+            {
+                type: ActionTypes.MessageBack,
+                title: 'Delete App',
+                value: 3,
+                text: 'delete',
+            },
+            {
+                type: ActionTypes.MessageBack,
+                title: 'Publish App',
+                value: 4,
+                text: 'publish',
+            },
+            {
+                type: ActionTypes.MessageBack,
+                title: 'Update app',
+                value: 5,
+                text: 'update',
+            }
+        ];
+            await this.sendWelcomeCard(context, cardActions);
+    }
+
+    async ListCardData(context) {
+        const cardActions = [
+            {
+                type: ActionTypes.MessageBack,
+                title: 'Home',
+                value: 1,
+                text: 'home',
+            },
+            {
+                type: ActionTypes.MessageBack,
+                title: 'List Apps',
+                value: 3,
+                text: 'listapp',
+            },
+            {
+                type: ActionTypes.MessageBack,
+                title: 'ListApp by ID',
+                value: 4,
+                text: 'app',
+            },
+            {
+                type: ActionTypes.MessageBack,
+                title: 'App based on manifest Id',
+                value: 5,
+                text: 'findapp',
+            },
+            {
+                type: ActionTypes.MessageBack,
+                title: 'App Status',
+                value: 6,
+                text: 'status',
+            },
+            {
+                type: ActionTypes.MessageBack,
+                title: 'List of bot',
+                value: 7,
+                text: 'bot',
+            }
+        ];
+        const card = CardFactory.heroCard(
+            'AppCatalog Sample',
+            '',
+            null,
+            cardActions
+        );
+        await context.sendActivity(MessageFactory.attachment(card));
+    }
+    async sendUpdateCard(context, cardActions) {
```

Comment:
Could you please check and remove if this is not used? Please do the same for other instances if any.